agent/config/config_unix_test.go

```diff
@@ -73,6 +73,7 @@ func TestConfigFromFile(t *testing.T) {
   "EngineAuthData": %s,
   "DataDir": "/var/run/ecs_agent",
   "TaskIAMRoleEnabled": true,
+  "TaskCPUMemLimit": true,
```

Comment:
(I can't comment on other lines because GitHub's diff is weird) Can you add an assertion to `TestConfigDefault` that verifies it's `DefaultEnabled`?

Reply:
Sure